people_tool/static/people_tool/js/services/AngularDRFService.js

```diff
@@ -0,0 +1,5 @@
+(function() {
```

Comment:
I don't believe we need this file in the people_tool. Is there a reason this is included?